sys/include/net/lorawan/hdr.h

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2016 Fundación Inria Chile
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_lw_hdr    LoRaWAN header
```

Comment:
group should be called `net_lorawan_hdr` to avoid confusion (we now have a lwmac protocol ;) )